qlib/FreetdsSqlUtil.qm

```diff
@@ -62,6 +62,25 @@ module FreetdsSqlUtil {
 
     @note This module requires the <a href="https://github.com/qorelanguage/module-sybase">freetds</a> binary module for communication with MS SQL Server and Sybase databases
 
+    @section freetds_sql_operations SQL Operations with the FreeTDS Driver
+
+    %FreetdsSqlUtil supports the following additional @ref select_option_hash "select option" when working with MS SQL Server databases:
+    - \c "tablehint": A single string or a list of strings with the following possible values:
+      - \c "fastfirstrow": Equivalent to SQL Server option \c "(FAST 1)"
+      - \c "holdlock": Makes shared locks more restrictive by holding them until the completion of a transaction (instead of releasing the shared lock as soon as the required table or data page is no longer needed, whether or not the transaction has been completed). The scan is performed with the same semantics as a transaction running at the \c "SERIALIZABLE" isolation level.  equivalent to \c "serializable"
+      - \c "nolock": Equivalent to \c "readuncommitted"
+      - \c "paglock": Takes shared page locks where a single shared table lock is normally taken
+      - \c "readcommitted": Specifies that a scan is performed with the same locking semantics as a transaction running at \c "READ COMMITTED" isolation level
+      - \c "readpast": Specifies that locked rows are skipped (read past)
+      - \c "readuncommitted": Specifies that dirty reads are allowed
+      - \c "repeatableread": Specifies that a scan is performed with the same locking semantics as a transaction running at \c "REPEATABLE READ" isolation level
+      - \c "rowlock": Specifies that a shared row lock is taken when a single shared page or table lock is normally taken
+      - \c "serializable": Makes shared locks more restrictive by holding them until the completion of a transaction (instead of releasing the shared lock as soon as the required table or data page is no longer needed, whether or not the transaction has been completed). The scan is performed with the same semantics as a transaction running at the \c "SERIALIZABLE" isolation level.  Equivalent to \c "holdlock".
+      - \c "tablock": Specifies that a shared lock is taken on the table held until the end-of-statement. If \c "holdlock" is also specified, the shared table lock is held until the end of the transaction.
+      - \c "tablockx": Specifies that an exclusive lock is taken on the table held until the end-of-statement or end-of-transaction.
+      - \c "updlock": Specifies that update locks instead of shared locks are taken while reading the table, and that they are held until the end-of-statement or end-of-transaction.
+      - \c "xlock": Specifies that exclusive locks should be taken and held until the end of transaction on all data processed by the statement. If specified with \c "paglock" or \c "tablock", the exclusive locks apply to the appropriate level of granularity.
+
```

Comment:
these are hints for MS SQL Server 2008. 2012 provides more hints.
And mainly - the most important one is missing: "INDEX (...)" - unfortunately it gets arguments too.

Query and Join hints are missing. Is it wanted feature to support only table hints?